tests/phpunit/elementor/includes/managers/test-image.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace Elementor\Testing\Includes\Managers;
+
+use Elementor\Images_Manager;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Test_Image extends Elementor_Test_Base {
+
+	public function create_image() {
+		$attachment_id = $this->_make_attachment( [
+			'file' => __DIR__ . '/mock/mock-image.png',
```

Comment:
Please move the mock image to `/tests/phpunit/resources`

Reply:
Done.